runtime/altair/src/xcm.rs

```diff
@@ -71,7 +71,7 @@ impl frame_support::traits::Contains<RuntimeCall> for SafeCallFilter {
 				| RuntimeCall::PolkadotXcm(
 					pallet_xcm::Call::limited_reserve_transfer_assets { .. }
 				) | RuntimeCall::XcmpQueue(..)
-				| RuntimeCall::DmpQueue(..)
+				| RuntimeCall::MessageQueue(..)
```

Comment:
I think at this point we could drop this safe call filter. Polkadot is doing the same for system parachains: https://github.com/polkadot-fellows/runtimes/pull/285

The only reason holding me back here (i.e. _could_ wording) is that we use default benchmarks.

Reply:
The default benchmarks are computed with the "required" hardware. So I guess we're in the same magnitude. We could compute them by ourselves too, but we are not sure if we want such precision now

Reply:
Wow, removing that would be great!